frontend/js/views/alert.js

```diff
@@ -105,26 +80,27 @@ define(
             TYPES: {
                 ERROR: {
                     title: "alert.error.title",
-                    className: "alert-error"
+                    severity: "alert-error"
                 },
                 FATAL: {
                     title: "alert.fatal.title",
                     className: "alert-error",
```

Comment:
Should probably also be:
```suggestion
                    severity: "alert-error",
```